app/templates/home.hbs

```diff
@@ -33,14 +33,9 @@
         <div class="sidebar">
           <p>Popular Tags</p>
           <div class="tag-list">
-            <a href="" class="tag-pill tag-default">programming</a>
-            <a href="" class="tag-pill tag-default">javascript</a>
-            <a href="" class="tag-pill tag-default">emberjs</a>
-            <a href="" class="tag-pill tag-default">angularjs</a>
-            <a href="" class="tag-pill tag-default">react</a>
-            <a href="" class="tag-pill tag-default">mean</a>
-            <a href="" class="tag-pill tag-default">node</a>
-            <a href="" class="tag-pill tag-default">rails</a>
+            {{#each model.popularTags as |tag|}}
```

Comment:
I think it is semantics.
I am not sure what the preferred way is.
Personally I think the direction Ember has been planning on moving to in a long time is removing Controllers altogether.
In some of my other projects I use router-actions instead of controller actions and in general avoid controllers if I can.